charts/incubator/wyoming-whisper/questions.yaml

```diff
@@ -0,0 +1,108 @@
+# Include{groups}
+portals: {}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+# Include{containerBasic}
+# Include{containerAdvanced}
+  - variable: wyomingWhisper
+    group: App Configuration
+    label: Whisper Configuration
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: model
+          label: Model
+          description: Whisper model that will be used for transcription.
+          schema:
+            type: string
+            required: true
+            default: tiny-int8
```

Comment:
```suggestion
        - variable: model
          label: Model
          description: Whisper model that will be used for transcription.
          schema:
            type: string
            required: true
            default: tiny-int8
            enum:
              - value: tiny-int8
                description: Tiny-int8
              - value: tiny
                description: Tiny
              - value: base-int8
                description: Base-int8
              - value: base
                description: Base
              - value: small-int8
                description: Small-int8
              - value: small
                description: Small
              - value: medium-int8
                description: Medium-int8
              - value: medium
                description: Medium
```

Reply:
I thought about this, but as new models are released this chart will need to be manually updated.

Reply:
you can add a custom blank option and basically if its set can show the custom option for users to add their custom input....take a peak at arksurvivalevolved in the incubator train for an example.